lib/index.ts

```diff
@@ -38,6 +44,16 @@ function buildDepTreeFromProjectJson(manifestFileContents: string, includeDev =
   return getDependencyTreeFromProjectJson(manifestFile, includeDev);
 }
 
+// TODO: Figure out what to do about devDeps
+function buildDepTreeFromProjectAssetsJson(manifestFileContents: string, targetFramework?: string): PkgTree {
+  if (!targetFramework) {
+    throw new Error('Missing required parameters for buildDepTreeFromProjectAssetsJson()');
```

Comment:
Since it's just one parameter, can you say `missing targetFramework for ...`

Reply:
Update error message :)